src/index.ts

```diff
@@ -109,7 +109,7 @@ export class PyTutorExtension
             src += '###############################\n';
           }
           console.log('code cell', cell_index);
-          src += `${cell.model.value.text}\n`;
+          src += `${cell.model.toJSON().source}\n`;
```

Comment:
Would the following also work?

```suggestion
          src += `${cell.model.sharedModel.source}\n`;
```

Reply:
It does work, thanks.